Danki2/Assets/Scripts/Abilities/Ability.cs

```diff
@@ -1,9 +1,32 @@
-﻿namespace Abilities
+﻿using UnityEngine;
+
+namespace Abilities
 {
-    public enum Ability
+    public abstract class AbilityContext
```

Comment:
Should this be abstract? If you want to use the ability builder you need to be able to pass in a context

Reply:
@nick-wilson95 not sure if you saw this comment?

Reply:
also, should the fields on the ability context have getters?